components/apps/ddm/scenes/Datasets.tsx

```diff
@@ -46,14 +47,16 @@ export default function Datasets(props: {
         {props.datasets
           .filter((dataset, i) => !props.search || dataset.name.includes(props.search))
           .map((dataset, i) => {
-            let progress = dataset.bytes_replicated.padded / dataset.bytes_total.padded / dataset.replication_quota;
+            // let progress = dataset.bytes_replicated.padded / dataset.bytes_total.padded / dataset.replication_quota;
```

Comment:
i think you meant to comment it back in

Reply:
Good catch... I've removed my test data in latest.